groups/bsl/bslstl/bslstl_optional.h

```diff
@@ -3076,6 +3069,15 @@ class optional<TYPE, false> : public std::optional<TYPE> {
         // 'original.value()' (of 'ANY_TYPE') converted to 'TYPE' otherwise.
         // 'original' is left in a valid but unspecified state.
 
+    template <class... ARGS>
+    explicit optional(bsl::in_place_t,
+                      BSLS_COMPILERFEATURES_FORWARD_REF(ARGS)...);
```

Comment:
Let's not use `BSLS_COMPILERFEATURES_FORWARD_REF` in the C++17 code - please use plain `&&` and `std::forward` in this section of code.

Reply:
oops, missed it. Also missed the docs here and in the pre c++17 version. Fixed all